src/components/ArticleHolder.jsx

```diff
@@ -0,0 +1,49 @@
+// import axios from 'axios';
+import {useState, useEffect} from 'react';
+// import { Routes, Route } from "react-router-dom";
+// import Article from './Article';
+
+
+
+function ArticleHolder () {
+    const [articles, setArticles] = useState([]);
```

Comment:
It would be great if we could have a loading state in here to give the user feedback when the data is being fetched 👍